NO-JIRA: Fix the "Enabling USB host passthrough" wrong procedure (#110064)
openshift/openshift-docs: modules/virt-enabling-usb-host-passthrough.adoc

----

. Open the `HyperConverged` CR in your default editor by running the following command:
+
[source,terminal,subs="attributes+"]
----
$ oc edit {HCOCliKind} kubevirt-hyperconverged -n {CNVNamespace}
----

. Add the required USB device to the `permittedHostDevices` stanza of the `HyperConvered` CR. The following example adds a device with vendor ID `045e` and product ID `07a5`:
+
[source,yaml,highlight=11..12,subs="attributes+"]
----
apiVersion: hco.kubevirt.io/v1beta1
kind: HyperConverged
metadata:
  name: kubevirt-hyperconverged
  namespace: {CNVNamespace}
spec:
  permittedHostDevices:
    usbHostDevices:
    - resourceName: kubevirt.io/peripherals
      selectors:
      - vendor: "045e"
        product: "07a5"
      - vendor: "062a"
        product: "4102"
      - vendor: "072f"
        product: "b100"
----
+
* `spec.permittedHostDevices` defines the host devices that have permission to be used in the cluster.
* `spec.permittedHostDevices.usbHostDevices` defines a list of available USB devices.

> * use `resourceName: deviceName` for each device you want to add and assign to the VM. In this example, the resource is bound to three devices, each of which is identified by `vendor` and `product` and is known as a `selector`.

Reviewer: ```suggestion
*  `spec.permittedHostDevices.usbHostDevices.resourceName` defines the USB device that you want to add and assign to the VM. In this example, the resource is bound to three devices, each of which is identified by `vendor` and `product` and is known as a `selector`.
```

> * `selectors` is a list of vendor-product pairs. Add your new USB device here.

Reviewer: ```suggestion

```

.Verification

> * Ensure that the KubeVirt custom resource contains the required USB devices:

Reviewer: Do we refer to the `KubeVirt` CR in the downstream documentation? I am confused because we edited the `HyperConverged` CR in the procedure so shouldn't we be checking the same CR in the Verification section?

Author: In that case, I prefer to remove the verification section. There is no point to set a field and then check that it's there.
